### PR TITLE
business email validation added some UI fixes and text fixes on send code component

### PR DESCRIPTION
**Pull request by kapildave07**
## Description

Business email validation added, some UI fixes and text fixes on send code component

## Changes

- handled editable fields on business email
- fix the validations
- update the UI for disabled
- fix some minor text

## Attached Links

1. Jira ticket: https://investax.atlassian.net/browse/IXSPD1-1419
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/pages/KYC/IndividualKycFormV2.tsx [hunk 1]
```diff
@@ -198,27 +202,16 @@ export default function IndividualKycFormV2() {
...
-        ...prevValues,
-        email: email || '',
-        firstName: firstName || '',
-        middleName: middleName || '',
-        lastName: lastName || '',
-      }))
+      setInitialValues(getInitialValues())
     }
-  }, [kyc?.individual])
+  }, [isPersonalVerified, kyc?.individual?.isEmailVerified, kyc?.individual?.secondaryContactDetails])
```
Comment (thi-investax) [inline src/pages/KYC/IndividualKycFormV2.tsx:214]: @kapildave07  could you please write comment for each if else condition. The logic is tricky i don' fully tunderstand

File: src/pages/KYC/IndividualKycFormV2.tsx [hunk 2]
```diff
@@ -247,19 +240,26 @@ export default function IndividualKycFormV2() {
...
     if (errors[key] || form.current.touched[key]) {
       validateValue(key, value)
     }
-  }
 
+    if (kyc?.individual && values[key] !== value) {
+      const kycValue = kyc.individual[key]
+      if (value.length === 0 || value !== kycValue) {
+        setFieldValue(key, value, true)
+      }
```
Comment (thi-investax) [inline src/pages/KYC/IndividualKycFormV2.tsx:252]: could you please comment the purpose of this code block @kapildave07 ? thanks

File: src/pages/KYC/IndividualKycFormV2.tsx [hunk 3]
```diff
@@ -247,19 +246,27 @@ export default function IndividualKycFormV2() {
...
+
     if (errors[key] || form.current.touched[key]) {
       validateValue(key, value)
     }
-  }
 
+    // Check if the KYC individual data exists and if the current value is different from the KYC individual data value
+    if (kyc?.individual && values[key] !== value) {
+      const kycValue = kyc.individual[key] // Get the current value from KYC data for comparison
+      if (value.length === 0 || value !== kycValue) {
```
Comment (thi-investax) [inline src/pages/KYC/IndividualKycFormV2.tsx:257]: what if value is undefined or null or something? Is value always a string?

Comment (kapildave07) [reply src/pages/KYC/IndividualKycFormV2.tsx:257]: @thi-investax we already handle null or undefined values reasonably well due to the use of optional chaining (kyc?.individual) and conditional checks (values[key] !== value). and this is only check if individual data exists.

**Pull request merged**